plugin/core/settings.py

```diff
@@ -149,9 +149,19 @@ def read_language_configs(client_config: dict) -> 'List[LanguageConfig]':
     return list(map(read_language_config, client_config.get("languages", [])))
 
 
+def read_commands(commands: 'List[Dict[str, Any]]') -> 'List[Command]':
+    validCommands = []
```

Comment:
style nitpick: please use snake_case variable names (so `valid_commands` instead of `validCommands`)